src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+@import "../base/base";
+@import "../abstracts/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: calc(#{$cell-dimensions + $gap + $padding} * 7);
+
+  &__day {
+    background-color: $cell-color;
+    font-size: $font;
+    width: $cell-dimensions;
+    height: $cell-dimensions;
+    box-sizing: border-box;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    transition: background-color 0.5s;
+  }
+
+  &__day:hover {
+    background-color: pink;
+    transform: $cell-animation;
+  }
+
+  @for $i from 1 through 31 {
+    &__day:nth-child(#{$i})::after {
+      content: "#{$i}";
+    }
+  }
+
+  // margin-left added based on a day of the week
+  @each $day, $i in mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6 {
```

Comment:
You can create and map variables for each day of the week, from `mon` to `sun`.

Reply:
README file explicitly says to use '@each' for that